wfe/web-front-end.go

```diff
@@ -1002,26 +989,34 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 	logEvent := wfe.populateRequestEvent(request)
 	defer wfe.logRequestDetails(&logEvent)
 
+	// Certificate paths are of one of two forms:
+	// 1. CertPath + sixteen hex digits (for serial-based cert URLs)
+	// 2. CertPath + token (for request ID based cert URLs)
```

Comment:
I'm not crazy about relying on being able to syntactically separate serial numbers and request tokens.  It would be cleaner to do something like:

1. `/acme-cert/cert/id/<requestID>`
2. `/acme-cert/cert/serial/<serial>`